langchain/chains/conversation/base.py

```diff
@@ -0,0 +1,34 @@
+"""Chain that carries on a conversation and calls an LLM."""
+from pydantic import BaseModel, Extra
+
+from langchain.chains.base import Memory
+from langchain.chains.conversation.memory import ConversationBufferMemory
+from langchain.chains.conversation.prompt import PROMPT
+from langchain.chains.llm import LLMChain
+from langchain.prompts.base import BasePromptTemplate
+
+
+class ConversationChain(LLMChain, BaseModel):
+    """Chain to have a conversation and load context from memory.
+
+    Example:
+        .. code-block:: python
+
+            from langchain import ConversationChain, OpenAI
+            conversation = ConversationChain(llm=OpenAI())
+    """
+
+    memory: Memory = ConversationBufferMemory()
```

Comment:
i would use `Field(default_factory = ConversationBufferMemory)` to avoid putting mutable defaults